test/libsolidity/semanticTests/structs/calldata/calldata_struct_with_nested_array_to_memory.sol

```diff
@@ -0,0 +1,19 @@
+pragma experimental ABIEncoderV2;
+
+contract C {
+    struct S {
+        uint128 p1;
+        uint256[][2] a;
+        uint32 p2;
+    }
+    function f(uint32 p1, S calldata c) external returns(uint32, uint128, uint256, uint256, uint32) {
+        S memory s = c;
+        assert(s.a[0][0] == c.a[0][0]);
+        assert(s.a[1][1] == c.a[1][1]);
```

Comment:
Can you also change something in `s` and assert that it's != from `c`?

Reply:
I added some changes and assertions